API/NoDocs/Microsoft.Extensions.Telemetry.Tracing/SamplerType.cs

```diff
@@ -0,0 +1,10 @@
+// Assembly 'Microsoft.Extensions.Telemetry'
+
+namespace Microsoft.Extensions.Telemetry.Tracing;
+
+public enum SamplerType
```

Comment:
Can we remove this? I think it is dead code without an OTel dependency for distributed trace configuration.

Reply:
Done.